test-e2e/panels/spectra.test.ts

```diff
@@ -127,33 +127,35 @@ test('2d spectrum', async ({ page }) => {
     await expect(Tabs.last()).toHaveText('1H,1H');
   });
   await test.step('Change 1H spectrum color', async () => {
+    await nmrium.page.click('_react=InternalTab[tabid="1H"]');
+
     // Check ColorIndicator initial color
     await expect(
-      nmrium.page.locator('_react=ColorIndicator[color.color="#7c2353"]'),
+      nmrium.page.locator('_react=ColorIndicator[color.color="#C10020" i]'),
```

Comment:
We also need to understand why this color changed. Moreover it is only composed of 5 symbols and color should be 6.

I would expect that we correct the code rather than fixing the test.

Reply:
@lpatiny 

the first color should be always red #C10020, i do not know how it was working before !!!!!!!

Reply:
Ok. And you know why there is not this `i` in `="#C10020" i]'),` ?

Reply:
I just add it in case it is in a small or capital case. no matter how I enter the value.

Reply:
I took the value directly from the color function https://github.com/cheminfo/nmrium/blob/84ea3e4ef8471d60a8bec508410771e3cd0331c1/src/data/utilities/getColor.ts#L2

Reply:
@lpatiny 

Can I know why it is not good to use it like that from your point of view?